### PR TITLE
Make ClockMock Tests\\ namespace matching less specific

### PR DESCRIPTION
**Pull request by teohhanhui**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.8+ |
| Fixed tickets | N/A |

### CODE REVIEW

**Comment by WouterJ** [comment]
Thanks @teohhanhui, it looks great this way!

**Comment by teohhanhui** [comment]
@WouterJ Heads up, the related PR symfony/symfony#18726 was merged into master (3.2) so perhaps this should be reverted?

File: components/phpunit_bridge.rst [hunk 1]
```diff
@@ -207,10 +207,10 @@ Troubleshooting
 ~~~~~~~~~~~~~~~
 
 The ``@group time-sensitive`` works "by convention" and assumes that the
-namespace of the tested class can be obtained just by removing the ``\Tests\``
+namespace of the tested class can be obtained just by removing the ``Tests\``
```
Comment (xabbuh) [inline components/phpunit_bridge.rst]: @teohhanhui This is the only change that must be revert, right? The fixes below are still valid if I don't miss anything.

Comment (teohhanhui) [inline components/phpunit_bridge.rst]: Yes.

**Pull request merged**